pkg/relayer/session/session.go

```diff
@@ -165,18 +166,21 @@ func (rs *relayerSessionsManager) mapBlockToSessionsToClaim(
 	defer rs.sessionsTreesMu.Unlock()
 
 	// Check if there are sessions that need to enter the claim/proof phase
-	// as their end block height was the one before the last committed block.
-	// Iterate over the sessionsTrees map to get the ones that end at a block height
-	// lower than the current block height.
+	// as grace period block height was the one before the last committed block.
+	// Iterate over the sessionsTrees map to get the ones that have a grace period
+	// ending at the current block height.
 	for endBlockHeight, sessionsTreesEndingAtBlockHeight := range rs.sessionsTrees {
 		// TODO_BLOCKER(@red-0ne): We need this to be == instead of <= because we don't want to keep sending
```

Comment:
Is this comment still relevant?

Reply:
Yes, until we implement what's on the `TODO_DISCUSS` below